service/index.js

```diff
@@ -29,7 +29,7 @@ var contentTypes = {".js": 'application/javascript', ".css": 'text/css'};
 
 // Default cache control policy
 app.use(function(req, res, next) {
-	res.set('Cache-Control', 'public, max-age='+one_day+', stale-while-revalidate='+one_week+', stale-if-error='+one_week);
+	res.set('Cache-Control', 'public, max-age='+one_week+', stale-while-revalidate='+one_week+', stale-if-error='+one_week);
```

Comment:
Hey, hold on a minute. Read the spec!

![image](https://cloud.githubusercontent.com/assets/1735391/8726217/cb905b56-2bd0-11e5-8ea3-1865db26728b.png)